build-support/bin/packages.py

```diff
@@ -713,13 +772,11 @@ def install_and_test_packages(version: str, *, extra_pip_args: list[str] | None
 # -----------------------------------------------------------------------------------------------
 
 
-def list_owners() -> None:
-    for package in PACKAGES:
-        if not package.exists_on_pypi():
-            print(f"The {package.name} package is new!  There are no owners yet.", file=sys.stderr)
-            continue
-        formatted_owners = "\n".join(sorted(package.owners()))
-        print(f"Owners of {package.name}:\n{formatted_owners}\n")
+def validate_roles() -> None:
+    # Validate that ownership sets are exactly what we expect, even for legacy packages
+    # that we no longer publish.  Effectively we're using release time as an opportunity
+    # to do wider security validation than is strictly necessary just for releasing.
+    PackageAccessValidator.validate_all()
```

Comment:
This only happens if you explicitly run the command `validate-roles`, right? Do you think we should do instead do it as part of `publish`? I fear no one will remember to run `validate-roles`.

Reply:
FYI my WIP PR to remove `release.sh` deleted some of those commands like `list-packages` because they seem unused so not worth maintaining.

Reply:
How about if I merge this and you can make this change in your WIP PR, and kill validate-roles there?

Reply:
Sure